inst/htmljs/animint.js

```diff
@@ -510,6 +510,14 @@ var animint = function (to_select, json_file) {
 
     Plots[p_name].scales = scales;
 
+    // draw background
+    svg.append("rect")
+      .attr("x", plotdim.xstart)
+      .attr("y", plotdim.ystart)
+      .attr("width", graph_width)
+      .attr("height", graph_height)
+      .attr("fill", p_info.panel_background.fill);
```

Comment:
you should use `.style` rather than `.attr` to set "fill" "stroke" "stroke-dasharray" and "stroke-width", as in the definition of geoms that make a `<rect>` https://github.com/tdhock/animint/blob/master/inst/htmljs/animint.js#L1093-L1096